tests/generic_relations_regress/tests.py

```diff
@@ -119,10 +119,12 @@ def test_target_model_len_zero(self):
         note = Note(note='Deserve a bonus', content_object=team1)
         note.save()
 
-    def test_target_model_nonzero_false(self):
-        """Test related to #13085"""
-        # __nonzero__() returns False -- This actually doesn't currently fail.
-        # This test validates that
+    def test_target_model_bool_false(self):
+        """
+        Saving a model with a GenericForeignKey to a model instance whose
+        __bool__ method returns False (Guild.__bool__() here) shouldn't fail
+        (#13085).
+        """
```

Comment:
@timgraham what about this docstring? It's mostly inspired by the `test_target_model_len_zero` one.